### PR TITLE
Prevent warning about DSTROOT being undefined.

### PR DESCRIPTION
**Pull request by silene**

### CODE REVIEW

File: tools/CoqMakefile.in [hunk 1]
```diff
@@ -200,7 +200,7 @@ TIMING_ARG=
 endif
 
 # Retro compatibility (DESTDIR is standard on Unix, DESTROOT is not)
-ifneq "$(DSTROOT)" ""
+ifdef DSTROOT
 DESTDIR := $(DSTROOT)
```
Comment (SkySkimmer) [inline tools/CoqMakefile.in]: Given the comment above, is it supposed to be `DSTROOT` or `DESTROOT`?

Comment (silene) [inline tools/CoqMakefile.in]: Good point, it seems the typo is in the comment.

**Review by gares — APPROVED** [review]

**Comment by maximedenes** [comment]
There is something odd with this PR. If I fetch it (even on a fresh git clone), the ref github sends me is not the right one. Don't touch it, I'll contact the github support.

**Comment by maximedenes** [comment]
@silene did you do anything special with this PR besides closing and reopening it?

**Comment by silene** [comment]
What happened is that I inadvertently pushed master to this branch (hence the automatic closing by github). Once I noticed the issue, I restored the branch to its older state and reopened the request. I guess this has confused github to no end.

**Comment by maximedenes** [comment]
I'll merge from your branch.

**Pull request merged**